WordPress/WordPressTest/NSManagedObject+Fixture.swift

```diff
@@ -0,0 +1,24 @@
+import CoreData
+
+extension NSManagedObject {
+
+    /// Loads the contents of any given JSON file into a new NSManagedObject instance.
+    ///
+    /// This helper method is useful for Unit Testing scenarios.
+    ///
+    /// - Parameters:
+    ///   - filename: The name of the JSON file to be loaded
+    ///   - context: The managed object context to use
+    /// - Returns: A new instance with property values of the given JSON file.
+    static func fixture(fromFile filename: String, context: NSManagedObjectContext) -> Self {
```

Comment:
Other nitpick: What do you think of using `insertInto context:`, like in [`init(entity:, insertInto:)`](https://developer.apple.com/documentation/coredata/nsmanagedobject)? I feel it would make it clear to readers what's the purpose of that parameter.

![image](https://user-images.githubusercontent.com/1218433/166187180-d24378a3-6e2b-434b-92a1-60b85b7e01f0.png)

Reply:
Oh definitely. That's much nicer. 👍